virtcontainers/device/drivers/vfio.go

```diff
@@ -97,17 +100,61 @@ func (device *VFIODevice) DeviceType() config.DeviceType {
 	return device.DevType
 }
 
+type VFIODevice_Type int32
+
+const (
+        VFIODEVICE_TYPE_VFIO_ERROR    VFIODevice_Type = 0
+        VFIODEVICE_TYPE_VFIO_NORMAL   VFIODevice_Type = 1
+        VFIODEVICE_TYPE_VFIO_MEDIATED VFIODevice_Type = 2
```

Comment:
No :snake: , yes :camel:     :smile:

Reply:
ok, thanks, I will modify it in the next review.